src/config/index.json

```diff
@@ -6,5 +6,6 @@
     "authorizedLabelLimit": 3
   },
   "collect": "#/connected",
-  "filterAppType": false
+  "filterAppType": false,
+  "enabledStoreParts": ["discover", "myapps"]
```

Comment:
`sections` looks like a more accurate name to me. `parts` feels too vague. What do you think ?

I don't think we should precise `store` in prop name as we are in store config file.

Reply:
Sections is already used in the app vocabulary as part of the apps listing :/

Reply:
I replaced it by `Pages` 👍